src/Microsoft.Identity.Web/Constants/Constants.cs

```diff
@@ -3,127 +3,127 @@
 
 namespace Microsoft.Identity.Web
 {
-    /// <summary>
-    /// General constants for Microsoft Identity Web.
-    /// </summary>
-    public static class Constants
-    {
-        /// <summary>
-        /// LoginHint.
-        /// Represents the preferred_username claim in the ID token.
-        /// </summary>
-        public const string LoginHint = "login_hint";
-
-        /// <summary>
-        /// DomainHint.
-        /// Determined by the tenant Id.
-        /// </summary>
-        public const string DomainHint = "domain_hint";
-
-        /// <summary>
-        /// Claims.
-        /// Determined from the signed-in user.
-        /// </summary>
-        public const string Claims = "claims";
-
-        /// <summary>
-        /// Bearer.
-        /// Predominant type of access token used with OAuth 2.0.
-        /// </summary>
-        public const string Bearer = "Bearer";
-
-        /// <summary>
-        /// AzureAd.
-        /// Configuration section name for AzureAd.
-        /// </summary>
-        public const string AzureAd = "AzureAd";
-
-        /// <summary>
-        /// AzureAdB2C.
-        /// Configuration section name for AzureAdB2C.
-        /// </summary>
-        public const string AzureAdB2C = "AzureAdB2C";
-
-        /// <summary>
-        /// Scope.
-        /// </summary>
-        public const string Scope = "scope";
-
-        /// <summary>
-        /// Policy for B2C user flows.
-        /// The name of the policy to check against a specific user flow.
-        /// </summary>
-        public const string Policy = "policy";
-
-        // IssuerMetadata
-        internal const string TenantDiscoveryEndpoint = "tenant_discovery_endpoint";
-        internal const string ApiVersion = "api-version";
-        internal const string Metadata = "metadata";
-
-        // Metadata
-        internal const string PreferredNetwork = "preferred_network";
-        internal const string PreferredCache = "preferred_cache";
-        internal const string Aliases = "aliases";
-
-        // AadIssuerValidator
+	/// <summary>
+	/// General constants for Microsoft Identity Web.
+	/// </summary>
+	public static class Constants
```

Comment:
There are many constants throughout the codebase. It may be worthwhile to rename this class to be more descriptive about the specific constants it holds.